go/common/enclave.go

```diff
@@ -109,9 +109,12 @@ type Enclave interface {
 	// GetBatch - retrieve a batch if existing within the enclave db.
 	GetBatch(hash L2BatchHash) (*ExtBatch, SystemError)
 
-	// GetBatchBySeqNo - retrieve batch by sequencer number if it's in the db
+	// GetBatchBySeqNo - retrieve batch by sequencer number if it's in the db.
 	GetBatchBySeqNo(seqNo uint64) (*ExtBatch, SystemError)
 
+	// GetRollupData - retrieve the first batch sequence and start time for a given rollup.
+	GetRollupData(internalRollup []byte) (*big.Int, *uint64, SystemError)
```

Comment:
I think it's better we return a "PublicRollupMetadata" struct (and a SystemError)